shared/test/middleware/helpers/test_storage_id.rb

```diff
@@ -0,0 +1,69 @@
+require_relative '../../test_helper'
+require_relative '../../../middleware/helpers/storage_id'
+
+class StorageIdTest < Minitest::Test
+  include SetupTest
```

Comment:
Turns out we had existing code to do the transaction, and all i needed was to include the correct module https://github.com/code-dot-org/code-dot-org/blob/5bf016f72d07907e72f1727a83feb778625496b1/shared/test/common_test_helper.rb#L52